### PR TITLE
Fixes Temporal Color Code failing in latest Fiji version

### PR DESCRIPTION
**Pull request by lacan**
Fixes makeLUTsArray function use the built-in method to get the list. 

The previous method started failing as of Fiji/ImageJ 2.1.0
See https://forum.image.sc/t/temporal-color-code-bug/41323?u=oburri

### CODE REVIEW

File: plugins/Scripts/Image/Hyperstacks/Temporal-Color_Code.ijm [hunk 1]
```diff
@@ -138,40 +138,8 @@ macro "Time-Lapse Color Coder" {
 }
 
 function makeLUTsArray() {
```
Comment (imagejan) [inline plugins/Scripts/Image/Hyperstacks/Temporal-Color_Code.ijm:140]: How about removing this function entirely, and changing line 146 to:

```
lutA = getList("LUTs");
```

Comment (lacan) [reply plugins/Scripts/Image/Hyperstacks/Temporal-Color_Code.ijm]: Works for me. I was keeping it for historical purposes :)

**Comment by imagejan** [comment]
Duplicate of #238.

**Comment by manerotoni** [comment]
I just would like to mention that these changes did not reach the main distribution. It stills has a non working call.

**Pull request closed without merging**